packages/core/src/Text/Text.spec.tsx

```diff
@@ -85,8 +85,15 @@ describe('Text', () => {
     expect(screen.getByText('Italic Text')).toHaveStyleRule('font-style', 'italic')
   })
 
-  test('textTransform', () => {
+  test('textDecoration', () => {
     render(<Text textDecoration='underline'>Underlined Text</Text>)
     expect(screen.getByText('Underlined Text')).toHaveStyleRule('text-decoration', 'underline')
   })
+
+  test('textTransform', () => {
```

Comment:
nb: might be valuable to test the other text transforms but not a huge deal either way.